backend/app/services/adapters/task_kinds.py

```
@@ -588,6 +588,227 @@ def get_user_tasks_lite(
+        member_counts = {}
+        if task_ids_for_members:
+            member_count_results = (
+                db.query(TaskMember.task_id, func.count(TaskMember.id).label("count"))
+                .filter(
+                    TaskMember.task_id.in_(task_ids_for_members),
+                    TaskMember.status == MemberStatus.ACTIVE,
+                )
+                .group_by(TaskMember.task_id)
+                .all()
+            )
+            member_counts = {row[0]: row[1] for row in member_count_results}
```

Ruff (0.14.10)

Local variable `member_counts` is assigned to but never used

Remove assignment to unused variable `member_counts`

(F841)